team_builder_react/src/utilities/API_Interface.js

```diff
@@ -0,0 +1,233 @@
+const axiosInstance = axios.create({
+    baseURL: 'http://127.0.0.1:8000'
+});
+
+/*
+    CheckListArrayToString takes an array of objects as a parameter and returns
+        a comma separated string where each value in the string is the name of a skill
+        in the passed in array.
+     
+    parameters: listOfSkills - an array of objects. Each element is { name: string, id: int }
+                    (as given by the CheckList component) and represents a
+                    technical skill that the build team should posses, if possible.
+    
+    returns: a comma separated string of skills
+*/
+const CheckListArrayToString = (listOfSkills) => {
+    var skillsString = "";
+
+    listOfSkills.forEach(skill => {
+        if (skillsString.length === 0){
+            skillsString = skill.name;
+        } else {
+            skillsString = skillsString + "," + skill.name;
+        }
+    });
+
+    return skillsString;
+}
+
+/*
+    SendPersonToBackEnd is an asynchronous function that takes an object as a
+        parameter and sends that object to the back-end as JSON in an HTTP
+        request. This function will post a new person and their information to
+        the back-end, which will then store it in the database.
+     
+    parameters: person - an object (or dictionary) to send to the back-end
+                ex: {
+                        firstName: '',
+                        lastName: '',
+                        skills: [], each element is { name: string, id: int }
+                        traits: [], each element is { name: string, score: int }
+                    }
+    
+    returns: if no server errors, a Promise that is the response from the server (https://axios-http.com/docs/res_schema).
+             if there was an error in the request, then the Axios error in a Promise.
+
+    What will actually be sent to the back-end:
+    {
+        "name_last": "Bar",
+        "name_first": "Foo",
+        "skills": "Python, Django",
+        "bpt_confidence": 0.1,
+        "bpt_delegator": 0.2,
+        "bpt_determination": 0.3,
+        "bpt_disruptor": 0.4,
+        "bpt_independence": 0.5,
+        "bpt_knowledge": 0.6,
+        "bpt_profitability": 0.7,
+        "bpt_relationship": 0.8,
+        "bpt_risk": 0.9,
+        "bpt_selling": 1.0
+    }
+*/
+export const SendPersonToBackEnd = async (person) => {
+    var objectToSend = {
+        name_last: person.lastName,
+        name_first: person.firstName,
+        skills: ""
+    };
+
+    // Add the skill name for each skill the user selected
+    objectToSend.skills = await CheckListArrayToString(person.skills);
+
+    person.traits.forEach(trait => {
+        objectToSend["bpt_" + trait.name] = trait.score;
+    })
+
+    try {
+        const response = await axiosInstance.post('/employees/', objectToSend);
```

Comment:
I don't think you need to wrap this in a try catch if you use .catch((err) => {//handle here}.

Reply:
I think I did that at first but the editor said that I should use try catch. I think it's since I'm using await here. If I took that out then we could use .catch(err). I think it's just styling at that point but I can make the change if there is some kind of performance difference or something.